src/Pages/Edit.elm

```diff
@@ -180,18 +180,18 @@ update msg model =
                             info
             in
             ( { model | info = newInfo }
-            , Effect.none
+            , Effect.fromCmd (Effect.promptToSave True)
             )
 
         Submit ->
```

Comment:
You should move it to `GotSubmit`, simply pressing the submit doesn't save it just yet